python/nano/test/onnx/pytorch/test_onnx.py

```diff
@@ -283,6 +284,45 @@ def forward(self, x1, x2):
         return {"intermediate": [x1,x2], "x":x, "output":output}
 
 
+class MultipleInputNet_multi_input_type(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.dense1 = nn.Linear(12, 1)
+        self.dense2 = nn.Linear(11, 1)
+        self.dense3 = nn.Linear(10, 1)
+        self.dense4 = nn.Linear(9, 1)
+        self.dense5 = nn.Linear(8, 1)
+        self.dense6 = nn.Linear(7, 1)
+        self.dense7 = nn.Linear(6, 1)
+
+    def forward(self, x_1: Dict, x_2: Tuple, x_3: List, bbox):
+        return self.dense1(x_1['x1']) + self.dense2(x_1['x2']) + self.dense3(x_2[0]) + self.dense4(x_2[1]) + \
+            self.dense5(x_3[0]) + self.dense6(x_3[1]) + self.dense7(bbox)
+    
+
+class MultipleInputNet_single_dict(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.dense1 = nn.Linear(12, 1)
+        self.dense2 = nn.Linear(11, 1)
+        self.dense3 = nn.Linear(10, 1)
+
+    def forward(self, x: Dict):
+        return self.dense1(x['x1']) + self.dense2(x['x2']) + self.dense3(x['x3'])
+    
+
+class MultipleInputNet_non_list_dict(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.dense1 = nn.Linear(12, 1)
+        self.dense2 = nn.Linear(11, 1)
+        self.dense3 = nn.Linear(10, 1)
+        self.dense4 = nn.Linear(9, 1)
+
+    def forward(self, bbox: torch.Tensor, x: Dict):
```

Comment:
what will happen if the forward is designed as `forward(self, x: Dict, bbox: torch.Tensor)`?

Reply:
Please refer to the comments in the next conversation for reference. 
And I think the last test case contains this situation.